chemprop/v2/cli/train.py

```diff
@@ -615,16 +628,17 @@ def main(args):
         test_loader = None
 
     model = MPNN(
-        mp_block,
-        agg,
-        readout_ffn,
-        None,
-        args.task_weights,
-        args.warmup_epochs,
-        args.num_lrs,
-        args.init_lr,
-        args.max_lr,
-        args.final_lr,
+        message_passing=mp_block,
+        agg=agg,
+        readout=readout_ffn,
+        batch_norm=True,
+        metrics=None,
+        w_t=args.task_weights,
+        warmup_epochs=args.warmup_epochs,
+        # num_lrs=args.num_lrs,  # this argument is currently commented out in MPNN.__init__()
+        init_lr=args.init_lr,
+        max_lr=args.max_lr,
+        final_lr=args.final_lr,
```

Comment:
delete keywords. We can't have every PR introducing stylistic changes of previously written code based on the author's preferences. Most of the library uses positional arguments, so it should reasonably maintain the convention where possible. I.e., here the keywords are exactly the same names as the variables.

Reply:
OK. The reason why I changed this is because I found since we were using positional arguments, we actually didn't realized that we misaligned the argument at all, starting from `batch_norm`. I can switch back to positional if you think that is proper.

Reply:
FWIW, I think we should have used a different convention, i.e. used keyword arguments when passing many (more than four?) arguments. Even when the keywords and variable names are the same, using positional arguments leads to (sometimes silent) errors when the arguments are misaligned.

Reply:
On the flip side, changing a variable name in a function definition can break a lot of client code if you're relying on keyword arguments.